projects/robots/robotis/darwin-op/worlds/.robotis_op3.wbproj

```diff
@@ -1,5 +1,5 @@
 Webots Project File version R2019b
-perspectives: 000000ff00000000fd00000003000000000000000000000000fc0100000001fb0000001a0044006f00630075006d0065006e0074006100740069006f006e0000000000ffffffff0000008700ffffff000000010000020c0000034bfc0200000001fb0000001400540065007800740045006400690074006f007201000000000000034b0000008b00ffffff0000000300000690000000b2fc0100000001fb0000000e0043006f006e0073006f006c00650100000000000006900000008700ffffff000004820000034b00000001000000020000000100000008fc00000000
+perspectives: 000000ff00000000fd00000003000000000000000000000000fc0100000001fb0000001a0044006f00630075006d0065006e0074006100740069006f006e0000000000ffffffff0000006900ffffff00000001000000f10000021efc0200000001fb0000001400540065007800740045006400690074006f007201000000000000021e0000008900ffffff00000003000003bf000000b2fc0100000001fb0000000e0043006f006e0073006f006c00650100000000000003bf0000006900ffffff000002cc0000021e00000001000000020000000100000008fc00000000
```

Comment:
It is better to not change the size of the Text editor, otherwise the robot is not fully visible in the scene because of the low 3D scene height.

Reply:
In that case, it is probably better to zoom a bit out in the 3D view, as with 'small resolution' the text editor takes quite a lot of space with the previous perspective (tested with 1920x1080 and 1024x768).